src/portal/src/app/project/tag-retention/tag-retention.component.ts

```diff
@@ -246,21 +299,26 @@ export class TagRetentionComponent implements OnInit {
     }
 
     clickAdd(rule) {
+        this.loadingRule = true;
         if (this.addRuleComponent.isAdd) {
             let retention: Retention = clone(this.retention);
             retention.rules.push(rule);
             if (!this.retentionId) {
                 this.tagRetentionService.createRetention(retention).subscribe(
                     response => {
                         this.refreshAfterCreatRetention();
+                        this.loadingRule = false;
```

Comment:
’this.loadingRule = false‘ should be placed inside function this.refreshAfterCreatRetention();

Reply:
fixed